pkg/roll/run_update.go

```diff
@@ -363,7 +363,7 @@ func (u *update) countHealthy(id rcf.ID, checks map[string]health.Result) (rcNod
 
 	ret.Desired = rcFields.ReplicasDesired
 
-	currentPods, err := rc.New(rcFields, u.kps, u.rcs, u.sched, u.labeler, u.logger).CurrentPods()
+	currentPods, err := rc.New(rcFields, u.kps, u.rcs, u.sched, u.labeler, u.logger, nil).CurrentPods()
```

Comment:
Should we inject an `Alerter` into the roll farm in this PR? Even if we don't use it except to pass through to this function, it will inevitably appear here later I'd guess.